Prepare release 0.9

RonjaPonja opened the pull request.

Commit:
Prepare release 0.9

hasegeli commented on setup.py:
```diff
@@ -17,6 +17,17 @@
+    # will fail anyway.
+    from pkg_resources import parse_version
+    from setuptools import setup, __version__
+    assert parse_version(__version__) >= parse_version('39.2.0')
```
`assert` is not the right thing in here.

Commit:
Move back to setup.py for another 10 years

Python people move so slow that using only a setup.cfg will break
pip installs for people until all python versions shipped before
very recently have died. Thanks Obama.

hasegeli commented on setup.py:
```diff
@@ -18,5 +18,59 @@
 
-setup()
+if PYTHON_VERSION < (3, 4):
+    from sys import exit
```
You can import this outside as well.

RonjaPonja replied:
Fixed
